shortfin/python/shortfin_apps/llm/server.py

```diff
@@ -68,6 +69,12 @@ def parse_args(argv):
     parser.add_argument(
         "--instances", type=int, default=1, help="Number of shortfin instances to run"
     )
+    parser.add_argument(
+        "--enable_multiple_hip_streams",
+        action=BooleanOptionalAction,
```

Comment:
Just use `argparse.BooleanOptionalAction` rather than including the additional import.